data_processing.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Register TFL flip angle maps to the MP2RAGE UNI1 scan ⏳
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spinal-cord-7t/coil-qc-code/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>